tests/ExternalPackageTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests;
+
+use Composer\InstalledVersions;
+use Illuminate\Http\Request;
+use MongoDB\Laravel\Tests\Models\User;
+use Spatie\QueryBuilder\AllowedFilter;
+use Spatie\QueryBuilder\AllowedSort;
+use Spatie\QueryBuilder\QueryBuilder;
+
+/**
+ * Test integration with external packages.
+ */
+class ExternalPackageTest extends TestCase
+{
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+
+        User::truncate();
+    }
+
+    public function testSpacieQueryBuilder(): void
+    {
+        if (! InstalledVersions::isInstalled('spatie/laravel-query-builder')) {
```

Comment:
Not critical, but do we need to check whether this is installed? I'd say that the test should fail if the dependency was removed instead of silently being skipped, as we want to maintain compatibility with the package.

Reply:
Indeed, and the package will always be present in dev.